nyc_taxis/index.json

```diff
@@ -80,10 +80,10 @@
         "scaling_factor": 100,
         "type": "scaled_float"
       },
-      "trip_distance": {
+      "trip_distance": {%- if trip_distance_type is defined %} {{ trip_distance_type | tojson }} {%- else %} {
```

Comment:
@IanHoang would appreciate your feedback on this solution: it turned out we don't need change in test procedure - the default one works just fine (it has aggregations over `trip_distrance`) but what we need it the mapping type change for `trip_distance`. Using workload params seems like one of the ways to achieve that (without copy/pasting the whole index).

Reply:
`workload-params.json`:
```
{
    "trip_distance_type": {
        "type": "unsigned_long"
    }
}
```

And use like that: `opensearch-benchmark execute_test --pipeline=benchmark-only --workload=nyc_taxis --target-hosts=localhost:9200 --workload-params=workload-params.json
`